Automattic/woocommerce-payments
# [Simplifying Deposits UI] Adds the next deposits table

> **Note** - I've marked this PR as Needs Review but I still plan on writing tests

---

Fixes #5696 

#### Changes proposed in this Pull Request

 - New Deposits Status Chip component
 - New Next Deposit table on the Payments overview page









This PR is adding the Next Deposit table to the new Deposits Overview section:

![image](https://user-images.githubusercontent.com/2275145/225824866-e2d68214-225b-4d59-b636-542a7b087b21.png)

As part of working on this table, I introduced a new `DepositsStatusChip` component which will be handy when developing the deposits history table as well as the next deposits.

This component takes the raw deposit status from Stripe and renders a status pill/chip:

```
<DepositStatusChip
     status={ status } // Can be paid, pending, estimated, failed, in_transit. Defaults to Estimated
     isCompact
/>
```

Note: the component maps paid and in_transit statuses to "Sent" and pending to "Processing".

<div style="width:100%">
  <img src="https://user-images.githubusercontent.com/2275145/225825783-f322c1f0-de9e-43de-bafa-722650d398ec.png" width="180" />
  <img src="https://user-images.githubusercontent.com/2275145/225826023-4e771814-af67-4d60-944a-3a02bdcbf10c.png" width="180" />
  <img src="https://user-images.githubusercontent.com/2275145/225826199-5499b9c8-b660-44e1-b9af-f5fc456c3f14.png" width="180" />
  <img src="https://user-images.githubusercontent.com/2275145/225826353-70a32741-5a68-4871-bd6e-bd4d01a9bad9.png" width="180" />
</div>

#### Testing instructions





1. Pull the changes and run `npm run build:deps`
2. Set `_wcpay_feature_simplify_deposits_ui` to `1` in the wp-options table
3. From WP Admin go to Payments to view the Overview page
4. Check the Next deposits tables loads as per the designs

-------------------


**Post merge**
